backend/app/Models/Todo.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Todo extends Model
+{
+    protected $fillable = ['title', 'status'];
+
+    /**
+     * 状態定義
+     */
+    const STATUS = [
+        1 => ['label' => '未着手', 'class' => 'bg-danger'],
+        2 => ['label' => '着手中', 'class' => 'bg-info'],
+        3 => ['label' => '完了', 'class' => 'bg-success'],
```

Comment:
名前をもう少しいい感じににしたいかな…

classとかだと PHPのクラスのことかもしれないと思ってしまうので(実際これはPHPのファイルだし)
```suggestion
        1 => ['label_text' => '未着手', 'html_class' => 'bg-danger'],
        2 => ['labe_textl' => '着手中', 'html_class' => 'bg-info'],
        3 => ['label_text' => '完了', 'html_class' => 'bg-success'],
```

みたいなかんじかな…

Reply:
この辺、こちらのチュートリアルの内容をほぼ引用しましたw
https://www.hypertextcandy.com/laravel-tutorial-todo-app-list-tasks/

classは確かにそうかもしれないですね。、